src/app/tracing/graph/cy-graph/cy-listeners.ts

```diff
@@ -82,6 +93,36 @@ export function addCyZoomListener(cy: Cy, onZoom: () => void): void {
     cy.on(CY_EVENT_ZOOM, onZoom);
 }
 
+export function addCyBoxZoomListerner(
+    cy: Cy,
+    onBoxZoom: (boxStartPosition: Position, boxEndPosition: Position) => void,
+): void {
+    let lastBoxStart: BoxStartProps | null = null;
+
```

Comment:
It would be even shorter to use only a `let boxZoomStartPosition: Position | null = null;` here.